jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -139,6 +139,12 @@ public WebServer updateWebServer(final UpdateWebServerRequest anUpdateWebServerC
                                      final User anUpdatingUser) {
         anUpdateWebServerCommand.validate();
 
+        WebServer originalWebServer=getWebServer(anUpdateWebServerCommand.getId());
+        if(!anUpdateWebServerCommand.getNewName().equalsIgnoreCase(originalWebServer.getName())&& !anUpdateWebServerCommand.getState().toStateLabel().equals(WebServerReachableState.WS_NEW.toStateLabel())){
+            throw new WebServerServiceException("Web Server "+originalWebServer.getName()+" is in "+originalWebServer.getState().toStateLabel()+" state, can only rename new web servers");
```

Comment:
Use MessageFormat since it's easier to read and cleaner

    throw new WebServerServiceException(MessageFormat.format("Web Server {0} is in {1} state, can only rename new web servers", 
                                                             originalWebServer.getName(), originalWebServer.getState().toStateLabel()));

Notice that I made this 2 lines instead of 1. The general norm in formatting code is to make lines that are too long to fit on screen (well in our case, the Github's review page). As a guide I set my Intellij IDEA to show a line (120 columns). Go to File -> Settings -> Editor -> Code Style -> Right Margin Columns = 120.